src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -103,6 +105,8 @@ export default class HaAutomationActionRow extends LitElement {
 
   @internalProperty() private _yamlMode = false;
 
+  @query("ha-yaml-editor", true) private _yamlEditor?: HaYamlEditor;
```

Comment:
```suggestion
  @query("ha-yaml-editor") private _yamlEditor?: HaYamlEditor;
```

I don't think we should change this?

Reply:
I took that over from `ha-automation-action-events.ts` and `ha-automation-action-services.ts`. There they are cached as well.

Reply:
Then they should be updated too since we rerender/remove them, it should not be cached.

Reply:
Just checked, `ha-automation-action-event` and `action-service` never remove the YAML editor so, that is good to cache.

Reply:
Caching now removed for all three.

Reply:
Dang, did not see your update. Will add the caching back in...